pkg/build/builder/sti.go

```diff
@@ -262,27 +261,27 @@ func (s *S2IBuilder) Build() error {
 			dockercfg.PushAuthType,
 		)
 		if authPresent {
-			glog.Infof("Using provided push secret for pushing %s image", pushTag)
+			glog.V(2).Infof("Using provided push secret for pushing %s image", pushTag)
 		} else {
-			glog.Infof("No push secret provided")
+			glog.V(2).Infof("No push secret provided")
 		}
-		glog.Infof("Pushing %s image ...", pushTag)
+		glog.V(0).Infof("Pushing %s image ...", pushTag)
 		if err := pushImage(s.dockerClient, pushTag, pushAuthConfig); err != nil {
 			// write extended error message to assist in problem resolution
 			msg := fmt.Sprintf("Failed to push image. Response from registry is: %v", err)
 			if authPresent {
-				glog.Infof("Registry server Address: %s", pushAuthConfig.ServerAddress)
-				glog.Infof("Registry server User Name: %s", pushAuthConfig.Username)
-				glog.Infof("Registry server Email: %s", pushAuthConfig.Email)
+				glog.V(3).Infof("Registry server Address: %s", pushAuthConfig.ServerAddress)
+				glog.V(3).Infof("Registry server User Name: %s", pushAuthConfig.Username)
+				glog.V(3).Infof("Registry server Email: %s", pushAuthConfig.Email)
```

Comment:
this only gets printed on a push failure, i think it should stay at a default-enabled level.